src/Microsoft.DotNet.Build.CloudTestTasks/AzureHelper.cs

```diff
@@ -0,0 +1,234 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Globalization;
+using System.Linq;
+using System.Net;
+using System.Security.Cryptography;
+using System.Text;
+using System.Text.RegularExpressions;
+using Microsoft.Build.Utilities;
+
+namespace Microsoft.DotNet.Build.CloudTestTasks
+{
+    public class AzureHelper
+    {
+        public const string STORAGE_API_VERSION = "2015-04-05";
+
+        public static string createContainerSASToken(string AccountName, string ContainerName, string key,
```

Comment:
Should capitalize "Create" and Pascal "Sas".

Parameters should be "accountName"